src/scss/components/_scroll-indicator.scss

```diff
@@ -1,6 +1,7 @@
 .scroll {
   background-color: $white;
   display: block;
+  margin-bottom:76px;
   padding: 20px;
```

Comment:
Please add some extra numbers like padding: 20px 0 0 0; other wise the padding is being added all around.

Reply:
@mfbarrera Is this change you requested so that the component matches the design? Otherwise, what you mean it's that the element should have a padding-top instead. Is that correct?

Reply:
Because it match the design